tests/lifecycle/test_case_test.go

```diff
@@ -172,6 +175,23 @@ func (s serviceLifecycleTestCase) execute(resourceGroup string) error {
 			return bErr
 		}
 	}
+
+	//Iterate through any child test cases, setting the instnace from this
+	//test case as the parent.
+	for i, test := range s.childTestCases {
+		test.parentService = &instance
+		var subTestName string
+		if test.description == "" {
+			subTestName = fmt.Sprintf("subtest-%v", i)
+		} else {
+			subTestName = strings.Replace(test.description, " ", "_", -1)
+		}
+		t.Run(subTestName, func(t *testing.T) {
+			tErr := test.execute(t, resourceGroup)
+			assert.Nil(t, tErr)
+		})
+	}
+
```

Comment:
Ooooh. You weren't passing `t` around as a way of failing tests. You were using it to launch additional subtests.

Reply:
I think this is ok as long as we're sure failures are always getting chalked up to the correct case.